context/app/main.py

```diff
@@ -18,6 +18,14 @@ def access_denied(e):
     return render_template('errors/403.html', types={}), 403
 
 
+def unauthorized(e):
```

Comment:
Why call this unauthorized if it's expired?

Reply:
"Unauthorized" is the name for the HTTP code... It seems to be caused by the globus credentials expiring, but conceivably it could have another cause that I haven't hit yet.